vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/client/AzureDevOpsClient.java

```diff
@@ -19,58 +19,109 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.List;
+import java.util.Set;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import org.apache.http.HttpEntity;
 import org.apache.http.client.methods.HttpRequestBase;
 import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.vividus.azure.devops.client.model.AddOperation;
+import org.vividus.azure.devops.client.model.PointsQuery;
+import org.vividus.azure.devops.client.model.TestPoint;
+import org.vividus.azure.devops.client.model.TestPointContainer;
+import org.vividus.azure.devops.client.model.TestResult;
+import org.vividus.azure.devops.client.model.TestRun;
 import org.vividus.azure.devops.configuration.AzureDevOpsExporterOptions;
 import org.vividus.http.HttpMethod;
 import org.vividus.http.HttpRequestBuilder;
 import org.vividus.http.client.IHttpClient;
+import org.vividus.util.json.JsonUtils;
 
 public class AzureDevOpsClient
 {
     private static final Logger LOGGER = LoggerFactory.getLogger(AzureDevOpsClient.class);
 
-    private static final String API_VERSION = "?api-version=6.1-preview.3";
+    private static final String API_VERSION_3 = "?api-version=6.1-preview.3";
```

Comment:
Any reason to use two previews?

Reply:
https://docs.microsoft.com/en-us/rest/api/azure/devops/test/points/get-points-by-query?view=azure-devops-rest-6.0 this API call is missing in 6.1-preview.3

Reply:
what about 7.1-preview.2

Reply:
For now I would prefer to use the main version (6.1) the same

Reply:
difficult to answer since we're not devops api developers, but this call exists in the latest (for now) API version 7.1-preview.2

Reply:
In this case Ivan's comment is valid